prov/gni/src/gnix_ep.c

```diff
@@ -551,30 +644,54 @@ int gnix_ep_open(struct fid_domain *domain, struct fi_info *info,
 	 * TODO, initialize vc hash table
 	 */
 	if (ep_priv->type == FI_EP_RDM) {
-		ep_priv->vc_hash_hndl = NULL;
 		ret = _gnix_cm_nic_alloc(domain_priv,
 					 &ep_priv->cm_nic);
 		if (ret != FI_SUCCESS)
 			goto err;
+
+		gnix_ht_attr.ht_initial_size = 64;     /* TODO: get from domain */
+		gnix_ht_attr.ht_maximum_size = 16384;  /* TODO: from domain */
+		gnix_ht_attr.ht_increase_step = 2;
+		gnix_ht_attr.ht_increase_type = GNIX_HT_INCREASE_MULT;
+		gnix_ht_attr.ht_collision_thresh = 500;
+		gnix_ht_attr.ht_hash_seed = 0xdeadbeefbeefdead;
```

Comment:
You might want to set  ht_internal_locking to something to prevent odd or difficult to reproduce behavior.